draft-ietf-rats-architecture.md

```diff
@@ -1317,3 +1317,160 @@ use beyond the period for which it deems the Attestation Result to remain
 `time(OP) - time(ER) < time(RX)-time(RG)`.
+
+# Appendix B: System Examples
+
+(Rough first draft to see if there is interest)
+
+## System Example with OS Vendor as Main Driver
+
+Three entities (companies / organizations) participate in this examples.
+
+The operating system vendor is at the center. It sets the architecture and the level of security achieved. It performs the following:
+* Generates large quantities of ECDSA key pairs to be used as per-device attestation keys
+* Signs the public keys using an attestation PKI and root that they control creating per-device certificates.
+* Securely sends the private keys and certificates to the device OEM in batches to be used in manufacturing
+* Authors the device SW that is the Attester and ships it to the OEMs. This is a part of the OS.
+* Publishes the root certificate and definition of the claims
+
+The device OEM does the manufacturing. It performs the following:
+* Physically makes the device to OS vendors security requirements
+* Installs the OS on the device
+* Personalizes each device with a unique private key and certificate supplied by the OS vendor
+
+The application vendor that implements some application with a back-end cloud service. It performs the following:
+* Author an app on the device that calls the attestation API to obtain Attestation Evidence
+* Convey the Attestation Evidence to the back-end cloud service over the network
+* Verify the Attestation Evidence and decided whether or not to allow application service
+
+This maps as follows in the RATS architecture:
+
+Endorser:
+
+: The operating system vendor is the endorser
+
+Endorsement:
+
+: The root certificate published by the endorser
+
+Attester:
+
+: The HW made by the device OEM and the SW made by the OS vendor
+
+Attestation Evidence:
+
+: A protocol message generated by the Attester on the request of the application that is then sent to the application vendor cloud service.
+
+Verifier and Verifier Owner:
+
+: are run by the application vendor cloud service
+
+Relying Party and Relying Party Owner:
+
+: are also the application vendor cloud service; since the Verify and Relying Part are the same the distinction between passport and background check doesn’t apply.
+
+Attestation Result:
+
+: non existent as the Verifier is the same as the Rely Party
+
+
+## System Example Made Up of Chip Vendor and IoT Device Maker
+
+Two entities (companies / organizations) participate in this example.
+
+The chip vendor that makes the chip. They also make the SW that is the Attester. It does the following:
+* Manufacturers the chip
+* Authors the SW that runs on the chip that is the Attester
+* Personalizes each chip with a unique 256-bit symmetric key and key ID
+* Also stores the 256-bit symmetric key in a well-secured database
+* Operates a web-based HMAC verification service the inputs to which are the key ID and HMAC to verify and the output is a yes or no on the verification
```

Comment:
I don't think that this level of detail about key generate belongs in the RATS architecture, even as an appendix.